src/uwtools/config.py

```diff
@@ -70,6 +73,17 @@ def __init__(self, config_path=None):
 
         self.config_path = config_path
 
+        #initialize logger with parent inheritance
+        if __name__ == '__main__':
+            _name = __name__
```

Comment:
Aha, ok - I think I lost track of the logic here, I was focusing on the class conditional example rather than cutting down to what we really meant to do, thanks

Reply:
Fixed to check directly against the logger directory